packages/core/src/env-set/oidc.ts

```diff
@@ -6,11 +6,11 @@ import inquirer from 'inquirer';
 
 import { noInquiry } from './parameters';
 
-const readPrivateKey = async (path: string): Promise<string> => {
+const readPrivateKey = async (): Promise<string> => {
```

Comment:
finally got what did you mean by removing useless path input  🤣

Reply:
hahah